method-info/winsorisation/example-data/winsorisation_expansion_input.csv

```diff
@@ -0,0 +1,11 @@
+reference,period,group,target_value,design_weight,l_value
+123456,202201,1001,50,100,3000
```

Comment:
quick question where did you get this data pr was it something you gathered/computed from the method code?

Reply:
Partly from the original method spec and partly from the testing alongside the method.  This is being reviewed by methodology.